### ISSUE TITLE
Very interesting, but need some richer examples

### ISSUE THREAD

**Issue by dartdog** [ScottFreeLLC/AlphaPy#6]
I think you need a few more ready to go and enriched examples preferably notebooks that can be run directly in a working directory. As currently set up the rather skinny examples cannot be run directly in place that is if I clone a copy of the repo and start up a one of the notebook examples... Also they seem pretty incomplete vs the full capabilities?.. I'm referring specifically to the two trading ones.. (have not looked at the others..)

Nonetheless I appreciate your work,, when I can I may be able to create something along this line to contribute back,, but it will take some time since the current examples are so sparse!

**Comment by mrconway** [comment]
Agreed, we'll add some rendered notebooks, as there is considerably more underneath the surface. Thanks for your comments. Spot on.

**Comment by taewookim** [comment]
any update on this?

**Comment by mrconway** [comment]
The next release will connect a model's output (e.g., probabilities) with a system. For example, use the probabilities generated by a model to determine trade entry and/or exit. You will still be able to run a stand-alone model or stand-alone system, but we are applying a methodology similar to the one in this paper:

[https://arxiv.org/abs/1711.04174](url)

We'll first try this on our existing ML algorithms, such as XGBoost. We do not yet support Keras (deep learning), as that is slated for two releases from now.

Regarding the examples, we'll be adding a cryptocurrency model and system using intraday BTC data for the next release. Sorry to say that we've been focused on more functionality rather than showcasing the full API with richer examples. This still needs attention.

Thanks for your inquiry.

**Comment by ConsultingSecurty** [comment]
any update on this?
I need a working example for tradingsystem and marketprediction.

**Comment by mrconway** [comment]
Noted
